lib/ReactViews/Workbench/Controls/DimensionSelectorSection.tsx

```diff
@@ -71,7 +71,7 @@ export default class DimensionSelectorSection extends React.Component<{
               {(typeof dim.selectedId === "undefined" ||
                 dim.allowUndefined) && (
                 <option key="__undefined__" value="">
-                  Not specified
+                  {dim.undefinedLabel ?? "Not specified"}
```

Comment:
You should probably i11n the "Not specified" here

Reply:
ah yep, good point

Reply:
done